src/status_im/contexts/settings/wallet/data_store.cljs

```diff
@@ -53,27 +34,30 @@
   "Updates keypairs' accounts to be fully operable based on specified key-uids and an optional operable condition.
 
   Parameters:
```

Comment:
should we remove doc strings and instead use malli?
cc @clauxx, @ilmotta, @mmilad75 wdyt? 🤔

Reply:
fully onboard with this